### PR TITLE
blastem: use Python 3 and remove unneeded resources

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/blastem.rb [hunk 1]
```diff
@@ -22,81 +22,42 @@ class Blastem < Formula
...
   depends_on "gettext" => :build
+  depends_on "imagemagick" => :build
   depends_on "jpeg" => :build
   depends_on "libpng" => :build
   depends_on "openjpeg" => :build
   depends_on "pkg-config" => :build
   depends_on "glew"
-  depends_on :macos # Due to Python 2
+  depends_on "pillow"
+  depends_on "python@3.9"
```
Comment (carlocab) [inline Formula/blastem.rb:32]: We're shipping new (runtime) dependencies, so let's bump the `revision` here.

Comment (danielnachun) [reply Formula/blastem.rb]: Actually I made a mistake - both `python@3.10` and `pillow` are build time dependencies.  I think in this case we don't need a revision bump?

**Comment by nandahkrishna** [comment]
> I think we need to notify the `blastem` developer that `xcf2png` from `xcftools` is unmaintained and broken, and that they can just use `imagemagick` to convert the XCF files to PNGs

I've sent them a message.

**Comment by danielnachun** [comment]
> > I think we need to notify the `blastem` developer that `xcf2png` from `xcftools` is unmaintained and broken, and that they can just use `imagemagick` to convert the XCF files to PNGs
> 
> I've sent them a message.

Thanks @nandahkrishna!

**Comment by danielnachun** [comment]
One last question - I think it might make sense for us to add `depends_on arch :x86_64` because this is an emulator, and porting emulators to new host architectures is a non-trivial task.  So I don't think we can expect this to support ARM64 without a very significant amount of work.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**